app/lib/import_export.rb

```diff
@@ -159,7 +159,9 @@ module ImportExport # rubocop:todo Metrics/ModuleLength
                       'Self-monitoring with public health supervision',
                       'Self-monitoring with delegated supervision',
                       'Self-observation'],
-    case_status: ['Confirmed', 'Probable', 'Suspect', 'Unknown', 'Not a Case']
+    case_status: ['Confirmed', 'Probable', 'Suspect', 'Unknown', 'Not a Case'],
+    lab_type: ['PCR', 'Antigen', 'Total Antibody', 'IgG Antibody', 'IgM Antibody', 'IgA Antibody', 'Other'],
+    result: %w[positive negative indeterminate other]
```

Comment:
Let's be consistent with these arrays so either using the `%w[...]` syntax or the same syntax that's used in all the other fields in this object.

Reply:
@kylietmo I think rubocop actually forces us to use different ones depending on the situation. If the array contains no strings with spaces, it'll want the `%w` version and if not, it'll want the regular version

Reply:
Ah, that's really annoying haha. Thanks Tim. I'll resolve this then.